src/map.cpp

```diff
@@ -4677,15 +4677,15 @@ static void process_vehicle_items( vehicle *cur_veh, int part )
     bool washing_machine_finished = false;
     if( washmachine_here ) {
         for( auto &n : cur_veh->get_items( part ) ) {
-            const int washing_time = MINUTES( 90 );
-            const int time_left = washing_time - n.age();
+            const time_duration washing_time = 2_hours;
```

Comment:
`2_hours` isn't equivalent to `MINUTES( 90 )`. Is that intentional?

Reply:
I don't know how to translate 90 minutes in to new for.

Reply:
> don't know how to translate 90 minutes in to new for.

`90_minutes` and most probably `1.5_hours` will also work

Reply:
90_minutes is ok. _hours take int so 1.5_hours is not ok.